src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -69,9 +72,17 @@ public static void Register()
         }
 
         /// <summary>
+        /// You can add predicates to this list and they will be evaluated before
+        /// logging the request. If *any* fail the request will not be logged.
+        /// </summary>
+        public static ConcurrentBag<Func<HttpContext, bool>> RequestFilterPredicates => _requestFilterPredicates;
```

Comment:
Looking at this vs. the simple property setter below, would we be able to get away with just a single predicate here (like `ShouldLogPostedFormData`? Exposing a public `ConcurrentBag` doesn't sit quite right (`AddRequestFilter()` wrapping it would be an improvement).